docs/modules/getting-started/pages/get-started-rest-api-with-java.adoc

```diff
@@ -0,0 +1,178 @@
+= Getting Started with REST API Using Java
+:description: This tutorial introduces you to enabling, running and using the REST API.
+
+:page-enterprise: true
+
+{description}
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Any supported JDK
+|
+xref:deploy:versioning-compatibility.adoc#supported-java-virtual-machines[Supported Java Virtual Machines]
+
+|Maven
+|link:https://maven.apache.org/download.cgi[Download Maven]
+
+|===
+
+== Set Up the Java Maven Project
+
+You need to set up a Java project that you can later execute to start a Hazelcast member with REST API. Add the following dependencies to your `pom.xml` file.
+
+[source,xml,subs="attributes+"]
+----
+<dependencies>
+    <dependency>
+        <groupId>com.hazelcast</groupId>
+        <artifactId>hazelcast-enterprise-rest</artifactId>
+        <version>{full-version}</version>
+    </dependency>
+    <dependency>
```

Comment:
rest module already depends on the module `hazelcast-enterprise` do we really need that dependency as well?

Reply:
Yes we need it since in the rest pom.xml, enterprise dependency is specified with a scope of provided
```
        <dependency>
            <groupId>com.hazelcast</groupId>
            <artifactId>hazelcast-enterprise</artifactId>
            <version>${project.version}</version>
            <scope>provided</scope>
        </dependency>
```